docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server and focuses on requests generated by users.
+
+Example:
+- datasource metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
```

Comment:
```suggestion
You can limit access to internal systems, such as you can deny _user_ access to the Grafana server. This feature does not affect traffic from the a Grafana user’s browser.
```

Reply:
Vet this for technical accuracy after my changes.

Reply:
That's not quite correct. Another hard section to nail down.

Perhaps:
`You can limit access to internal systems, such that a user through (for example) data source metric queries is unable to reach systems that the Grafana server itself would be able to reach. This feature does not affect traffic from a Grafana user's browser.`

Reply:
How about: 
You can limit access to internal systems so that a user through data source metric queries is unable to reach systems that the Grafana server itself would be able to reach. This feature does not affect traffic from a Grafana user's browser.